x-pack/filebeat/input/azureblobstorage/scheduler.go

```diff
@@ -93,16 +92,30 @@ func (s *scheduler) schedule(ctx context.Context) error {
 }
 
 func (s *scheduler) scheduleOnce(ctx context.Context) error {
+	defer s.limiter.wait()
```

Comment:
Why is this a wait and not an acquire?

Reply:
We acquire the limiter when we do a job, which is defined below. When the scheduling of all the jobs in a current poll interval is done we release the limiter. This statement was previously residing in the schedule method, which did not seem right, since in the gcs input it resides here, in scheduleOnce.

Reply:
OK, thanks for clarifying.